deepchem/models/tensorgraph/models/graph_models.py

```diff
@@ -1149,13 +1149,24 @@ def predict_on_generator(self, generator, transformers=[]):
 
 import warnings
 
+TENSORGRAPH_DEPRECATION = "{} is deprecated and has been renamed to GraphConvModel and will be removed in DeepChem 3.0."
```

Comment:
This is really nice! Good design choice :)

Reply:
That has a bug with the second title though. Fixing now.

Reply:
Done.